core/src/test/java/org/itsallcode/openfasttrace/core/importer/TestImporterFactoryLoader.java

```diff
@@ -78,10 +80,10 @@ void beforeEach()
     }
 
     @Test
-    void testNoFactoryRegisteredThrowsException()
+    void testNoFactoryRegisteredReturnsNoImporter()
     {
         simulateFactories();
-        assertThrows(ImporterException.class, () -> this.loader.getImporterFactory(this.file));
+        assertEquals(Optional.empty(),this.loader.getImporterFactory(this.file));
```

Comment:
```suggestion
        assertTrue(this.loader.getImporterFactory(this.file).isEmpty());
```
Besser❓

Reply:
At least also good :-). Worked in proposed changes.